src/nvim/ops.h

```diff
@@ -3,14 +3,8 @@
 
 #include <stdbool.h>
 
-#include "nvim/ascii.h"
-#include "nvim/eval/typval.h"
 #include "nvim/ex_cmds_defs.h"  // for exarg_T
-#include "nvim/extmark.h"
-#include "nvim/macros.h"
-#include "nvim/normal.h"  // for MotionType and oparg_T
-#include "nvim/os/time.h"
```

Comment:
Why remove so many headers?

Reply:
I need to remove all non-defs headers according to https://github.com/neovim/neovim/issues/6371 . I thought this would be fine since it compiles fine.